alibi/utils/lang_model.py

```diff
@@ -33,6 +33,14 @@
 class LanguageModel(abc.ABC):
     SUBWORD_PREFIX = ''
 
+    # We don't type transformers objects here as it would likely require digging into
+    # some private base classes which may change in the future and cause breaking changes.
+    model: Any
+    tokenizer: Any
```

Comment:
Perhaps at some point we need to think more about what tokenizers we actually support, and type accordingly? I've been typing with `PreTrainedTokenizerBase` in the Alibi Detect config work as we instantiate the tokenizer with `AutoTokenizer.from_pretrainer`, but maybe this needs more thought...